### DISCUSSION TITLE
"include(libigl)"  is not supported any more

### DISCUSSION THREAD

**Discussion by Aref-Hemati** [libigl/libigl#2049] [Q&A] [answered]
I'm attempting to use an academic project that used Libigl but is no longer supported by its developer and my issue was not answered by them. The issue is related to using the deprecated  "include" method in their CMake project files to build Libigl during building the project.
Could I ask you, please check out this issue and guide me if you know how I can resolve it?

https://github.com/n1ckfg/SynDraw/issues/1#issue-1251685802

**Comment by jdumas** [comment, marked as answer]
This [SynDraw](https://github.com/n1ckfg/SynDraw) repository already comes with an old version of libigl (`SynDraw/libigl`). I'd suggest compiling the project with the bundled version directly.

**Reply by Aref-Hemati** [reply]
Thank you, Jérémie. yes, that's correct, and the problem is about libigl version. (I also made a mistake)

Originally this issue was about SynDraw in lnria repo I thought it was the same as SynDraw repo in Github. SynDraw in lnria repo does not have libigl folder, I use the last library version which is not compatible with it. This GitHub repo uses the right (old) version of libigl and works perfectly.
